### ISSUE TITLE
[amp-social-share] Buttons do not have an accessible name: Improve a11y of amp-social-share

### ISSUE THREAD

**Issue by eduardogoncalves** [Automattic/newspack-theme#1014]
When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers.

This issue is related to ampproject/amphtml#29563 but I think this should be fixed when writing `amp-social-share` tag on HTML template.

![image](https://user-images.githubusercontent.com/4308648/89457305-2e669480-d733-11ea-8f8e-c76ace87cdfb.png)

**Comment by eduardogoncalves** [comment]
Hi @laurelfulford :) do you think we can fix this at Newspack side or it's better to fix at Jetpack?

**Comment by laurelfulford** [comment]
Hi @eduardogoncalves! Thanks for flagging me on this -- I think it does make more sense to approach on the Jetpack side. The theme doesn't do anything special to the buttons, just uses their out-of-the-box markup for AMP that Jetpack provides. The PR you've started in the Jetpack repo will fix it on any Newspack sites :) 

I'm going to close this issue to defer to the work in Jetpack, but just let me know if you have any questions about the above -- thanks!
